src/index.js

```diff
@@ -94,6 +104,9 @@ const SwipeableViews = React.createClass({
     this.isScroll = undefined;
   },
   handleTouchMove(event) {
+    if (this.props.disabled)
```

Comment:
This extra check is not good for perfs.
I think that I would be better
To not listen to the event.